AUTHORS.md

```diff
@@ -13,6 +13,9 @@
 * Tim Keil, tim.keil@uni-muenster.de
     * second order derivatives for parameters
 
+* Linus Balicki, linus.balicki@ovgu.de
+    * subspace accelerated dominant pole algorithm in algorithms.samdp
+
```

Comment:
Looking below, it seems alphabetic ordering is preferred. (Also, pushing another commit might help with the testing.)